.travis.yml

```diff
@@ -20,7 +20,7 @@ before-install:
 
 install:
   - npm install -g dredd@4.0.0
-  - pip3 install -r requirements/tests.txt
+  - pip3 install mypy -r requirements/tests.txt
```

Comment:
This should be added to requirements/tests.txt

Reply:
Agreed.  I just wanted to quickly demonstrate the value of adding the tests.